src/redismodule.h

```diff
@@ -171,6 +171,7 @@ This flag should not be used directly by the module.
 #define REDISMODULE_NOTIFY_KEY_MISS (1<<11)   /* m (Note: This one is excluded from REDISMODULE_NOTIFY_ALL on purpose) */
 #define REDISMODULE_NOTIFY_LOADED (1<<12)     /* module only key space notification, indicate a key loaded from rdb */
 #define REDISMODULE_NOTIFY_MODULE (1<<13)     /* d, module key space notification */
+#define REDISMODULE_NOTIFY_UNLINK (1<<14)     /* module only key space notification, indicate a key released from db */
```

Comment:
I think REMOVED is a better term, "UNLINK" can be related to the UNLINK command
```suggestion
#define REDISMODULE_NOTIFY_REMOVED (1<<14)     /* module only key space notification, indicate a key is being removed from the db */
```